static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -91,8 +91,14 @@ const setupPlayer = (
         getAdConsentState(thirdPartyTrackingAdConsent) !== false;
     const disableRelatedVideos = !config.get('switches.youtubeRelatedVideos');
     // relatedChannels needs to be an array, as per YouTube's IFrame Embed Config API
-    const relatedChannels =
-        !disableRelatedVideos && channelId ? [channelId] : [];
+    const relatedChannels = [];
+    /**
+     * There's an issue with relatedChannels where
+     * if we pass a populated array no related videos are
+     * shown. Therefore for the time being we will pass an
+     * empty array.
+     */
+    // const relatedChannels = !disableRelatedVideos && channelId ? [channelId] : [];
```

Comment:
Great catch! Shall we add a Trello card to remind us to fix this?

Reply:
Yep good shout, I'll add a card to the board.